resources/views/index.blade.php

```diff
@@ -14,6 +14,7 @@
     <link rel="stylesheet" type="text/css" href="angular/account/register.css">
     <link rel="stylesheet" type="text/css" href="angular/about/about.css">
     <link rel="stylesheet" type="text/css" href="angular/admin/privacy.css">
+    <link rel="stylesheet" type="text/css" href="angular/home/welcome.css">
```

Comment:
Sure.  We use gulp at work, but I pretty much only use it, don't know how to set it up or really how it works.